src/zcbenchmarks.cpp

```diff
@@ -333,12 +334,12 @@ double benchmark_increment_note_witnesses(size_t nTxs)
         auto note = GetNote(*pzcashParams, sk, wtx, 0, 1);
         auto nullifier = note.nullifier(sk);
 
-        mapNoteData_t noteData;
+        mapSproutNoteData_t noteData;
```

Comment:
Are we adding benchmarks for sapling Sapling notes?

Reply:
I think this would be a good idea, but I would prefer to do that as another PR. I created #3395 for this.